doc/api/n-api.md

```diff
@@ -118,6 +118,88 @@ must make use exclusively of N-API by restricting itself to using
 and by checking, for all external libraries that it uses, that the external
 library makes ABI stability guarantees similar to N-API.
 
+## Building
+
+Unlike modules written in JavaScript, developing and deploying Node native
+addons using N-API requires an additional set of tools. Besides the basic
+tools required to develop for Node.js, the native addon developer requires a
+toolchain that can compile C and C++ code into a binary. In addition,
+depending upon how the native addon is deployed, the *user* of the native
+addon will also need to have a C/C++ toolchain installed.
+
+For Linux developers, the necessary C/C++ toolchain packages are readily
+available. For many developers, the [LLVM](https://llvm.org) compiler
+infrastructure is a good choice.
+
+For Mac developers, [Xcode](https://developer.apple.com/xcode/) offers all
+the required compiler tools. However, it is not necessary to install the
+entire Xcode IDE. The following command installs the necessary toolchain:
+
+```bash
+xcode-select --install
+```
+
+For Windows developers, [Visual Studio](https://visualstudio.microsoft.com)
+offers all the required compiler tools. However, it is not necessary to
+install the entire Visual Studio IDE. The following command installs the
+necessary toolchain:
+
+```bash
+npm install --global --production windows-build-tools
+```
+
+The sections below describe the additional tools available for developing
+and deploying Node native addons.
+
+### Build tools
+
+> Both the tools listed here require that *users* of the native
+> addon have a C/C++ toolchain installed in order to successfully install
+> the native addon.
```

Comment:
Why are these lines quoted?

Reply:
I was attempting to call the reader's attention to this point. But I see the technique is not used anywhere else in the document and I've removed the quote.